CliFx.Tests/Services/CommandSchemaResolverTests.cs

```diff
@@ -62,37 +64,192 @@ private static IEnumerable<TestCaseData> GetTestCases_GetCommandSchemas_Negative
 
             yield return new TestCaseData(new object[]
             {
-                new[] {typeof(NonImplementedCommand)}
+                new[] { typeof(NonImplementedCommand) }
             });
 
             yield return new TestCaseData(new object[]
             {
-                new[] {typeof(NonAnnotatedCommand)}
+                new[] { typeof(NonAnnotatedCommand) }
             });
 
             yield return new TestCaseData(new object[]
             {
-                new[] {typeof(DuplicateOptionNamesCommand)}
+                new[] { typeof(DuplicateOptionNamesCommand) }
             });
 
             yield return new TestCaseData(new object[]
             {
-                new[] {typeof(DuplicateOptionShortNamesCommand)}
+                new[] { typeof(DuplicateOptionShortNamesCommand) }
             });
 
             yield return new TestCaseData(new object[]
             {
-                new[] {typeof(ExceptionCommand), typeof(CommandExceptionCommand)}
+                new[] { typeof(ExceptionCommand), typeof(CommandExceptionCommand) }
             });
         }
 
+        private static IEnumerable<TestCaseData> GetTestCases_GetTargetCommandSchema_Positive()
+        {
+            yield return new TestCaseData(
+                new List<CommandSchema>
```

Comment:
I suggest using array initializer syntax instead, for less verbosity, i.e.
`new[]` instead of `new List<CommandSchema>`